components/Layout/SettingsPage.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <v-container>
+    <v-row v-if="pageTitle">
+      <v-col>
+        <v-row class="mx-0 justify-space-between">
+          <h2 class="text-h6 mb-2">
+            {{ $t(pageTitle) }}
+          </h2>
+          <div class="ml-a">
+            <slot name="actions" />
+          </div>
+        </v-row>
+      </v-col>
+    </v-row>
+    <v-row>
+      <slot name="setting-content" />
```

Comment:
Could maybe just be named `content` ? There's no other slot in this component to confuse it with, so shorted is better, I think.

Reply:
Yeah that's probably a better name

Reply:
Updated